test/msw-api/setup-server/scenarios/on-unhandled-request/error.test.ts

```diff
@@ -17,7 +17,13 @@ afterAll(() => server.close())
 test('errors on unhandled request when using the "error" value', async () => {
   const getResponse = () => fetch('https://test.mswjs.io')
 
-  await expect(getResponse()).rejects.toThrow(
-    'request to https://test.mswjs.io/ failed, reason: [MSW] Error: captured a GET https://test.mswjs.io/ request without a corresponding request handler.',
-  )
+  await expect(getResponse()).rejects.toThrowErrorMatchingInlineSnapshot(`
+          "request to https://test.mswjs.io/ failed, reason: [MSW] Error: captured a GET https://test.mswjs.io/ request without a corresponding request handler.
+
+            If you wish to intercept this request, consider creating a request handler for it:
+
+            rest.get(\\"/\\", (req, res, ctx) => {
```

Comment:
Please assert the entire suggestion, including request handler URL. We need to make sure we are suggesting a code that is copy-pastable for the exact warning/error scenario.

```diff
-rest.get(\\"/\\",
+rest.get('https://test.mswjs.io',
```

Reply:
I'm going to have to research how to do this, as `jest` appears to be automatically escaping things (or maybe that's happening elsewhere?).